apm-dist/src/main/assembly/binary.xml

```diff
@@ -51,6 +51,7 @@
                 <include>official_analysis.oal</include>
                 <include>gateways.yml</include>
                 <include>service-apdex-threshold.yml</include>
+                <include>no_upstream_real_address_agent.yml</include>

```

Comment:
Don't use a new file. This is a very simple config, please add a field to tracr receiver module provider. 
Only If the content is complex and usually changed, we need a separated file. This fix is not that case.

This config is nearly static, nearly no one should change.